docs/modules/ROOT/pages/cluster-monitor.adoc

```diff
@@ -1,6 +1,6 @@
 = Cluster Monitor
 
-If running with the Couchbase Cluster Monitor enabled, it will output the status of its checkers to Prometheus like the following:
+Couchbase cluster monitor is a core component of the CMOS that provides cluster-level information via a Prometheus endpoint. If Couchbase cluster monitor is enabled, it will output the status of its checkers to Prometheus like the following:
```

Comment:
```suggestion
Couchbase cluster monitor is a core component of CMOS that provides cluster-level information via a Prometheus endpoint.
If Couchbase cluster monitor is enabled, it will output the status of its checkers to Prometheus like the following:
```